source/src/protocols/protein_mpnn/ProteinMPNNProbabilitiesMetric.cc

```diff
@@ -147,6 +148,8 @@ ProteinMPNNProbabilitiesMetric::parse_my_tag(
 		}
 	}
 
+	deterministic_flag_ = tag->getOption< bool >( "deterministic_flag", deterministic_flag_ );
```

Comment:
Is there a reason to name the user-facing option `deterministic_flag` versus something like simply `deterministic`?

Reply:
It's mainly geared to match the `ProteinMPNNMover` option, which called it `deterministic_flag`:
https://docs.rosettacommons.org/docs/latest/scripting_documentation/RosettaScripts/xsd/mover_ProteinMPNNMover_type